examples/assert_lockfile_up_to_date/assert_lockfile_frozen.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+const fs = require('fs');
+const child_process  = require('child_process');
+const readline = require('readline');
+
+const FIX_COMMAND = 'bazel run //examples/assert_lockfile_up_to_date:update_pnpm_lockfile'
+
+// https://github.com/pnpm/pnpm/blob/8fa95fd868d1afac6f7454ac17f77f8a017d2110/packages/headless/src/index.ts#L166
+const OUTDATED_LOCKFILE_ERR_CODE = "ERR_PNPM_OUTDATED_LOCKFILE"
+
+// pnpm does not offer a no-install method to assert that a lockfile
```

Comment:
would be nice to file this upstream, then point to that issue here, so there's some chance of unwinding the hack later (and to learn if anyone else has a nicer hack)